internal/controller/config/config.go

```diff
@@ -358,3 +361,15 @@ func (config *ProgressiveConfig) GetChildStatusAssessmentSchedule(kind string) (
 	}
 	return AssessmentSchedule{}, fmt.Errorf("No Assessment Schedule found for kind %q", kind)
 }
+
+// GetAnalysisRunTimeout parses the GlobalConfig for the analysisRunTimeout string to convert into a time.Duration
+func (config *ProgressiveConfig) GetAnalysisRunTimeout() (time.Duration, error) {
+
+	analysisRunTimeout, err := strconv.Atoi(config.AnalysisRunTimeout)
+	if err != nil {
+		return 0, fmt.Errorf("invalid analysisRunTimeout value: %w", err)
+	}
+
+	return time.Duration(analysisRunTimeout) * time.Second, nil
```

Comment:
sorry, just thought of this - I think a typical thing to do is to have a default constant in the code if it's not configured. Can we set ours to 15 or 20 minutes?

Reply:
Got it, so we should both set the default in our manifests to 20 minutes and also return 20 minutes if its unset here.

Reply:
thanks, did you test it not being set just in case?

Reply:
I'm thinking you'll get an error in your Atoi conversion if it's an empty string

Reply:
That's probably the case, let me change the way this checks the string then.